packages/common/src/types.ts

```diff
@@ -944,6 +944,12 @@ export interface HostStatMemory {
   usedPercentage: number;
 }
 
+export type Sensors = {
+  temp1_input: number;
```

Comment:
I would follow the format of the rest of the types:
```
export type CpuTemperature = {
  currentTemperature: number;
  maxTemperature?: number;
  minTemperature?: number;
};
```

Note that I added the `?` because the value could be undefined, but adapt this code to your needs.

Reply:
Finally deleted this type.